### PR TITLE
 [PyTest #33 - Extract] Rework tests setup in order not to assume hardcoded paths

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title

### CODE REVIEW

**Comment by s0undt3ch** [comment]
:/ currently broken on windows

**Comment by s0undt3ch** [comment]
Issues fixed.
@dwoz This needs a review from you.

**Review by dwoz — APPROVED** [review]

File: tests/integration/master/test_clear_funcs.py [hunk 1]
```diff
@@ -15,85 +16,78 @@
...
+
+
+class ConfigMixin:
+    @classmethod
+    def setUpClass(cls):
+        cls.master_config = AdaptedConfigurationTestCaseMixin.get_config("master")
+        cls.minion_config = AdaptedConfigurationTestCaseMixin.get_temp_config(
+            "minion",
+            id="root",
+            transport="zeromq",
```
Comment (dwoz) [inline tests/integration/master/test_clear_funcs.py:29]: The transport needs to change if we're running the tcp transport tests.

https://github.com/saltstack/salt/pull/57188/files#diff-e0fc5514059fe3cbc849239025bf09f3R198

**Review by dwoz — CHANGES_REQUESTED** [review]

File: tests/integration/master/test_clear_funcs.py (hunk 1 above)
Comment (s0undt3ch) [reply tests/integration/master/test_clear_funcs.py]: Fixed. Thanks!

**Comment by s0undt3ch** [comment]
re-run all

**Review by dwoz — APPROVED** [review]

**Pull request merged**